CMakeLists.txt

```diff
@@ -78,6 +78,7 @@ option(WITH_DLT_LOGSTORAGE_CTRL_UDEV "PROTOTYPE! Set to ON to build logstorage c
 option(WITH_DLT_USE_IPv6      "Set to ON for IPv6 support"                                                       ON)
 option(WITH_DLT_KPI           "Set to ON to build src/kpi binaries"                                              OFF)
 option(WITH_DLT_FATAL_LOG_TRAP "Set to ON to enable DLT_LOG_FATAL trap(trigger segv inside dlt-user library)"    OFF)
+option(WITH_UDP_CONNECTION     "Set to ON to enable dlt UDP multicast SUPPORT"      ON)
```

Comment:
Add a status message in bottom of this file.

Reply:
Added